issue_processing/jira_issue_processing.py

```diff
@@ -67,15 +72,20 @@ def run():
     persons = load_csv(__srcdir_csv)
     # 2) re-format the issues
     issues = parse_xml(issues, persons)
-    # 3) update user data with Codeface database
+    # 3) load issue information via api
+    if not args.skip_history:
+        load_issue_via_api(issues, persons, __conf)
```

Comment:
Please pass the actual issue-tracker URL here, not the entire configuration object `__conf` as it is not needed here actually.